src/operators.jl

```diff
@@ -104,6 +104,18 @@ function reverse! end
     return g
 end
 
+"""
+    permute!(g, p)
+
+Permute the vertices of graph `g` in-place, according to permutation `p`. No checking is done to verify that p is a permutation.
+"""
+function permute! end
```

Comment:
this should probably not be a new function, just a new method for the existing `Base.permute!` function